tests/integration/defs/perf/nvdf.py

```
@@ -0,0 +1,578 @@
+            },
+        },
+        "size": 3000,
+    }
+    json_data = json.dumps(json_data)
+
+    data_list = []
+    try:
+        res = query(json_data, project_name)
+        if res:
+            data = res.json()
+            for data in data["hits"]["hits"]:
```

Ruff (0.14.1)

Loop control variable `data` overrides iterable it iterates

(B020)